### PR TITLE
r1.4

### LINKED ISSUE #159: Bad URL for User stories and Basic API test cases link in kyc-match-API-Readiness-Checklist.md  and in kyc-fill-in-API-Readiness-Checklist.md
**Problem description**
Bad URL for User stories and Basic API test cases link in **kyc-match-API-Readiness-Checklist.md**  and in **kyc-fill-in-API-Readiness-Checklist.md**

https://github.com/camaraproject/KnowYourCustomer/blob/r1.3/documentation/API_documentation/documentation/API_documentation/KnowYourCustomer_User_Story.md
https://github.com/camaraproject/KnowYourCustomer/blob/r1.3/documentation/API_documentation/code/Test_definitions/kyc-match.feature

**Expected behavior**
remove _**documentation/API_documentation/**_ from the URL because twice

### PR DESCRIPTION
**Pull request by fernandopradocabrillo**
#### What type of PR is this?

Add one of the following kinds:
* correction
* subproject management
* tests

#### What this PR does / why we need it:
Preparation for public release r1.4. It includes updates and fixes for test plan to be fully compliant with version v0.2.1 and a correction in the API Readiness checklist.

#### Which issue(s) this PR fixes:

Fixes #159

### CODE REVIEW

**Review by GillesInnov35 — APPROVED** [review]
Thanks a lot Fernando

File: documentation/API_documentation/kyc-fill-in-API-Readiness-Checklist.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Checklist for kyc-fill-in 0.2.0 in release r1.3
 |  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |  Y   |      |
 |  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |  Y   |      |
 |  5 | API documentation                            |   M   |         M         |    M    |    M   |  Y   | Embedded documentation into API spec - [link](/code/API_definitions/kyc-fill-in.yaml) |
-|  6 | User stories                                 |   O   |         O         |    O    |    M   |  Y   | [link](documentation/API_documentation/KnowYourCustomer_User_Story.md) |
+|  6 | User stories                                 |   O   |         O         |    O    |    M   |  Y   | [link](KnowYourCustomer_User_Story.md) |
 |  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |  Y   | [link](code/Test_definitions/kyc-fill-in.feature) |
```
Comment (hdamker) [inline documentation/API_documentation/kyc-fill-in-API-Readiness-Checklist.md:13]: The relative link in same directory would work, but we suggest in new template to use links relative to the repo base:

```suggestion
|  6 | User stories                                 |   O   |         O         |    O    |    M   |  Y   | [link](/documentation/API_documentation/KnowYourCustomer_User_Story.md) |
|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |  Y   | [link](/code/Test_definitions/kyc-fill-in.feature) |
```

File: documentation/API_documentation/kyc-match-API-Readiness-Checklist.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Checklist for kyc-match 0.2.1 in release r1.3
 |  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |  Y   |      |
 |  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |  Y   |      |
 |  5 | API documentation                            |   M   |         M         |    M    |    M   |  Y   | Embedded documentation into API spec - [link](/code/API_definitions/kyc-match.yaml) |
-|  6 | User stories                                 |   O   |         O         |    O    |    M   |  Y   | [link](documentation/API_documentation/KnowYourCustomer_User_Story.md) |
+|  6 | User stories                                 |   O   |         O         |    O    |    M   |  Y   | [link](KnowYourCustomer_User_Story.md) |
 |  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |  Y   | [link](code/Test_definitions/kyc-match.feature)  |
```
Comment (hdamker) [inline documentation/API_documentation/kyc-match-API-Readiness-Checklist.md:13]: The relative link in same directory would work, but we suggest in new template to use links relative to the repo base:

```suggestion
|  6 | User stories                                 |   O   |         O         |    O    |    M   |  Y   | [link](/documentation/API_documentation/KnowYourCustomer_User_Story.md) |
|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |  Y   | [link](/code/Test_definitions/kyc-match.feature) |
```

**Review by GillesInnov35 — DISMISSED** [review]
Thanks

**Review by ToshiWakayama-KDDI — DISMISSED** [review]
Thank you, @fernandopradocabrillo .

**Comment by fernandopradocabrillo** [comment]
Hi @hdamker, @camaraproject/release-management_maintainers is it okey if we merge this PR? Thanks!

**Comment by hdamker** [comment]
Sorry, I was ooo last week. Have created now the review issue in RM.

File: CHANGELOG.md [hunk 3]
```diff
@@ -16,6 +17,49 @@ The below sections record the changes for each API version in each release as fo
...
+
+**kyc-match v0.2.1 is the first initial version for the CAMARA KnowYourCustomer Match API**
+
+- API definition **with inline documentation**:
+    - OpenAPI [YAML spec file](https://github.com/camaraproject/KnowYourCustomer/blob/r1.4/code/API_definitions/kyc-match.yaml)
+    - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/KnowYourCustomer/r1.4/code/API_definitions/kyc-match.yaml&nocors)
+    - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/KnowYourCustomer/r1.4/code/API_definitions/kyc-match.yaml&nocors)
+
+**Main changes**
+- Update test plan and remove unneeded scenario
```
Comment (hdamker) [inline CHANGELOG.md:42]:
```suggestion
- Update test plan and remove unneeded scenario
- There are no changes to the API definitions and API version numbers compared to r1.3
```

**Review by hdamker — DISMISSED** [review]
Looks from release management perspective.

One suggestion for clarity, which you can take or leave.

**Comment by fernandopradocabrillo** [comment]
> Looks from release management perspective.
> 
> One suggestion for clarity, which you can take or leave.

I'll take it, I think it provides detail

**Review by hdamker — APPROVED** [review]

**Review by ToshiWakayama-KDDI — APPROVED** [review]
LGTM.  Thank you.

**Review by GillesInnov35 — APPROVED** [review]
Thanks

**Pull request merged**